src/overmap_ui.cpp

```diff
@@ -808,7 +850,7 @@ void draw( const catacurses::window &w, const catacurses::window &wbar, const tr
         print_hint( "TOGGLE_BLINKING", uistate.overmap_blinking ? c_pink : c_magenta );
         print_hint( "TOGGLE_OVERLAYS", show_overlays ? c_pink : c_magenta );
         print_hint( "TOGGLE_LAND_USE_CODES", uistate.overmap_land_use_codes ? c_pink : c_magenta );
-        print_hint( "TOGGLE_CITY_LABELS", uistate.overmap_show_city_labels ? c_pink : c_magenta );
+        print_hint( "TOGGLE_LOCATION_LABELS", uistate.overmap_show_city_labels ? c_pink : c_magenta );
```

Comment:
There is no such keybinding named `TOGGLE_LOCATION_LABELS` in https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/raw/keybindings.json and it is not used anywhere in the code, so we need to return keybinding identifier back or add it to aforementioned file (and also use it everywhere in the code instead of `TOGGLE_CITY_LABELS`).

![image](https://user-images.githubusercontent.com/16213433/54638132-59e3d300-4a9b-11e9-8ca9-880353b79cf3.png)